examples/bert-loses-patience/test_run_glue_with_pabee.py

```diff
@@ -20,6 +20,7 @@ def get_setup_file():
 
 
 class PabeeTests(TestCasePlus):
+    @require_torch_non_multigpu_but_fix_me
```

Comment:
this breaks if multigpu? 

should the decorator be called `require_torch_single_gpu?`

Reply:
We don't know, that's why it has "but_fix_me" - I just added it to all tests in examples, and now we need to go over all of them, review and either make each test work under multi-gpu or designate it as non_multi_gpu.

wrt naming let's discuss it in normal comments, since once resolved these get hidden.